WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -24,6 +24,13 @@ final class SettingsViewController: UIViewController {
         return ServiceLocator.stores.sessionManager.defaultAccount?.displayName ?? String()
     }
 
+    /// Main Site's Name
+    ///
+    private var siteName: String {
+        let nameAsString = ServiceLocator.stores.sessionManager.defaultSite?.name as String?
+        return nameAsString ?? String()
```

Comment:
Concern: Should we provide a non-empty default?

Reply:
But what can be the reason for missing `defaultSite`? If it's not a common case - empty title is fine.